bundle/Command/LegacyInitCommand.php

```diff
@@ -90,4 +91,46 @@ protected function createDirectories(InputInterface $input, OutputInterface $out
 EOT
         );
     }
+
+    protected function updateComposeJson(OutputInterface $output)
+    {
+        $updateComposerJson = false;
+        $composerJson = json_decode(file_get_contents('composer.json'), true);
+        if ($composerJson === null) {
+            $output->writeln("Unable to parse composer.json");
+            return;
+        }
+
+        if (!array_key_exists('legacy-scripts', $composerJson['scripts'])){
+            $legacy_scripts = [
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installAssets',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::installLegacyBundlesExtensions',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::generateAutoloads',
+                'eZ\\Bundle\\EzPublishLegacyBundle\\Composer\\ScriptHandler::symlinkLegacyFiles'
+            ];
+            $composerJson['scripts'] = array_merge(['legacy-scripts' => $legacy_scripts], $composerJson['scripts']);
+            $updateComposerJson = true;
+        }
+
+        if (!array_key_exists('symfony-scripts', $composerJson['scripts'])) {
+            $composerJson['scripts']['symfony-scripts'] = ['@legacy-scripts'];
+            $updateComposerJson = true;
+        } elseif (!in_array('@legacy-scripts', $composerJson['scripts']['symfony-scripts'])) {
+            $symfonyScripts = $composerJson['scripts']['symfony-scripts'];
+            $offset = array_search('@php bin/console assetic:dump', $symfonyScripts);
```

Comment:
it has not always used dump command: https://github.com/ezsystems/ezplatform/blob/1.13/composer.json#L76